src/labels/CustomLabels.labels

```diff
@@ -2233,6 +2241,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>error when a batch is missing a name</shortDescription>
         <value>Batch Name is a required field.</value>
     </labels>
+    <labels>
+        <fullName>bdiErrorDataImportCustomSettings</fullName>
+        <categories>Data Importer</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>error when updating data import custom settings</shortDescription>
+        <value>There was an error attempting to update data import custom settings.</value>
```

Comment:
Update to be, "There was an error attempting to update the Custom Settings named Data Import Settings."

When error is exposed to user, please also append the System Error at the end.